bin/stress-test/src/main.rs

```diff
@@ -141,7 +130,8 @@ async fn seed_store(data_directory: PathBuf, num_accounts: usize) {
     };
 
     // start generating blocks
-    let genesis_header = genesis_state.into_block().unwrap();
+    let database_filepath = data_directory.join(STORE_FILENAME);
```

Comment:
One alternative is to expose the `DataDirectory` structure I added in the store.